src/map.rs

```diff
@@ -1,20 +1,19 @@
 use crate::iter::*;
 use crate::node::*;
 use crate::raw::*;
+use core::borrow::Borrow;
+use core::fmt::{self, Debug, Formatter};
+use core::hash::{BuildHasher, Hash, Hasher};
+use core::iter::FromIterator;
+use core::sync::atomic::{AtomicIsize, AtomicUsize, Ordering};
 use crossbeam_epoch::{self as epoch, Atomic, Guard, Owned, Shared};
-use std::borrow::Borrow;
+#[cfg(feature = "std")]
 use std::collections::hash_map::RandomState;
-use std::fmt::{self, Debug, Formatter};
-use std::hash::{BuildHasher, Hash, Hasher};
-use std::iter::FromIterator;
-use std::sync::{
-    atomic::{AtomicIsize, AtomicUsize, Ordering},
-    Once,
-};
+use std::sync::Once;
 
 macro_rules! isize_bits {
     () => {
-        std::mem::size_of::<isize>() * 8
+        core::mem::size_of::<isize>() * 8
     };
 }
```

Comment:
Side note: since `core::mem::size_of` is a `const fn`, couldn't this be a `cons`?

Reply:
Oh, you're right, it probably could (and should).